packages/sdk-js/src/GrowthBook.ts

```diff
@@ -404,6 +443,34 @@ export class GrowthBook<
     return this._ctx.experiments || [];
   }
 
+  public getTrackedFeatures(): Record<string, string> {
```

Comment:
are these too many options?
- trackedExperiments (exists, but no public CRUD)
- trackedExperimentHashes (preferred)

- blockedExperiments (not used by edge, but nice to have?)
- blockedExperimentHashes (used by edge)